### DISCUSSION TITLE
Connected Glass

### DISCUSSION THREAD

**Discussion by Alastor-Kaneki** [BedrockTweaks/Files#321] [Ideas]
Just a simple pack that makes it so glass is connected to other glass. The outer glass has borders where it isn't connected to other glass and no borders where it's connected to other glass. This should also work on panes.

**Comment by SquatchHunter** [comment]
not currently possible,
post block flattening we can look into it again

**Reply by Alastor-Kaneki** [reply]
How long would you predict block flattening will be a thing?

**Reply by Alastor-Kaneki** [reply]
Also, what I was trying to say is a behavior pack that checks what the glass is connected to

**Reply by SquatchHunter** [reply]
we have no idea
and even as a BP it wouldnt be feasible and likely cause lag

**Comment by SquatchHunter** [comment]
duplicate of #112
